docker/services_base.yaml

```diff
@@ -17,18 +17,9 @@ version: "3.4"
 
 services:
   services_container:
-    image: pdo_services
-    build:
```

Comment:
+1 to remove this and consolidate the build
(the other way around of using just docker-compose worked as well)

Reply:
again. I dont think this is a solution.
or maybe its a solution that we'll end up changing again in a couple weeks.